VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,36 @@
+# Video 5: Open Governance
+Hello everyone! Thank you for joining us in today's video. Today, we will talk about Open Governance and why AsyncAPI is committed to this model. 
+
+Let's get started!
+
+## What is Open Governance
+Open Governance is a model of management based on transparency, collaboration, and ethics which results in clearer accountability and management. Driven by the participation of all the people that make up a project, company, or institution, it promotes a democratic system with inclusive growth.
+
+Open source has become an essential model for the success of open government initiatives. Both share many of the same premises. Developers share information, ideas, and code to create more innovative software solutions, collectively and individually. This scalable and flexible software allows anyone with the source code to modify, improve, and redistribute it for greater accessibility and reuse. Open-source software works with peer-to-peer production and mass collaboration principles, creating a more sustainable software development process for end-users. 
+
+The sharing of ideas and community-developed software has driven creative, scientific, and technological advancement in many industries: education, government, law, health, technology, and more. This movement creates an opportunity for community members to collaborate, share, build, and help each other achieve common goals. In this case, through source code.
+
+## Why AsyncAPI is committed to an Open Governance model
+AsyncAPI is committed to democratic management based on two principles: horizontality in decision making and transparency in project management. From the beginning of the specification, AsyncAPI has relied on collective effort and group identity. The project focus on a governance and decision-making system in which values come first. The means are as necessary as the goal. Responsibilities are shared, understanding that a community can only enjoy success jointly.
```

Comment:
`The project focus on a governance and decision-making system in which values come first` - what do you mean?

Reply:
That values come before everything else, and they are the basis for any type of decision within the project.

Reply:
but what values you have in mind?

Reply:
I am referring to horizontality and transparency as the values I am talking about above.  If it is not clear, we can delete the sentence 😅

Reply:
oh, I somehow did not see a link :) maybe remove or improve to get this "link"